.github/workflows/interop.yml

```diff
@@ -0,0 +1,52 @@
+name: Interoperability Testing
+on:
+  pull_request:
+  push:
+    branches:
+      - unstable
+
+jobs:
+  run-multidim-interop:
+    name: Run multidimensional interoperability tests
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/checkout@v3
+        with:
+          #repository: libp2p/test-plans
+          repository: status-im/libp2p-test-plans
+          path: test-plans
+          ref: nimsupport
+          submodules: true
+          fetch-depth: 0
+
+      - name: Build image
+        run: >
+          cd test-plans/multidim-interop/nim/v1.0 &&
+          make commitSha=$GITHUB_SHA image_name=nim-libp2p-head
+
+      - name: Create ping-version.json
+        run: >
+          echo """
+            {
+                "id": "nim-libp2p-head",
+                "containerImageID": "nim-libp2p-head",
+                "transports": [
+                    "tcp",
+                    "ws"
```

Comment:
Nice! Lets you have a single file. Don't forget to update this when you add new capabilities :)